helpers/auth.py

```diff
@@ -63,6 +63,9 @@ def setup_webdriver(self):
         options.add_argument('--ignore-certificate-errors')
         options.add_argument('--no-sandbox')
         options.add_argument('--disable-dev-shm-usage')
+        if environ['CHROMEDRIVER_DIR'] not in environ['PATH']:
```

Comment:
I don't know when but github action runners started to override the path variable, which caused the selenium to not find the driver path by default.

This code should only execute in that case

Reply:
add a note above the line for explanation